app/api/roll.py

```
@@ -27,22 +27,22 @@
 
 @router.post("/", response_model=RollResponse)
 @limiter.limit("30/minute")
-def roll_dice(
+async def roll_dice(
     request: Request,
```

Ruff (0.14.14)

Unused function argument: `request`

(ARG001)